src/backend/aspen/app/app.py

```diff
@@ -22,24 +23,28 @@
 else:
     aspen_config = DockerComposeConfig()
 
+deployment = os.getenv("DEPLOYMENT_STAGE")
+
 # We should be able to allow this in all environments and only alert on prod.
 # Init as early as possible to catch more
 sentry_sdk.init(
-    aspen_config.SENTRY_URL,
+    dsn=aspen_config.SENTRY_URL,
+    integrations=[FlaskIntegration()],
     # Set traces_sample_rate to 1.0 to capture 100%
     # of transactions for performance monitoring.
     # We recommend adjusting this value in production.
     traces_sample_rate=1.0,
 )
+sentry_sdk.set_user(
+    {"id": os.getenv("EC2_INSTANCE_ID"), "deployment_stage": deployment}
+)
```

Comment:
I think the user is supposed to be a website user. Maybe this would be better to include near here:
https://github.com/chanzuckerberg/aspen/blob/main/src/backend/aspen/app/app.py#L81